lib/file/widgets/file_tile.dart

```diff
@@ -22,22 +22,23 @@ class FileTile extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return FancyCachedBuilder<File>.handleLoading(
-      controller: fileId.controller,
-      builder: (context, file, isFetching) {
+    return EntityBuilder<File>(
+      id: fileId,
+      builder: handleEdgeCases((context, file, _) {
```

Comment:
We should handle loading manually with a `ListTile` of `FancyText`s as the current loading UI is pretty ugly.